src/deserialization.hpp

```diff
@@ -70,17 +71,6 @@ class ConcreteTensorProtoDeserializator {
     }
 };
 
-enum class ExtractChoice {
```

Comment:
We should have head towards direction where we don't need to include all frontentds in one file. Instead of Moving Extractor template to validation utils and including it here, put it in separate file, then move specialization in appropriate *_utils.hpp files (kfs/tfs etc).